desk/app/hits.hoon

```diff
@@ -118,6 +118,7 @@
       [%~.~ %gossip %source ~]
     ::
     ::  neighbours listen for new hits from us
+    ~&  >  "hits: received subscription at /gossip/source"
```

Comment:
This was introduced while merging `bm/frontend` into `main`; I think we can just leave this for now and I'll pick it up in another PR to remove some testing/debugging stuff.